tests/chainer_tests/functions_tests/loss_tests/test_triplet.py

```diff
@@ -21,11 +21,31 @@
 class TestTriplet(unittest.TestCase):
 
     def setUp(self):
-        # Sample differentiable inputs
-        eps = 1e-3
         if self.dtype == numpy.float16:
             eps = 1e-2
+            self.check_forward_options = {'rtol': 5e-3, 'atol': 5e-3}
+            self.check_backward_options = {
+                'eps': eps, 'rtol': 5e-2, 'atol': 5e-2}
+            self.check_double_backward_options = {
+                'eps': eps, 'rtol': 5e-2, 'atol': 5e-2}
+        elif self.dtype == numpy.float32:
+            eps = 1e-3
+            self.check_forward_options = {'rtol': 1e-4, 'atol': 1e-4}
+            self.check_backward_options = {
+                'eps': eps, 'rtol': 5e-4, 'atol': 5e-4}
+            self.check_double_backward_options = {
+                'eps': eps, 'rtol': 1e-3, 'atol': 1e-3}
+        elif self.dtype == numpy.float64:
+            eps = 1e-3
+            self.check_forward_options = {'rtol': 1e-4, 'atol': 1e-4}
+            self.check_backward_options = {
+                'eps': eps, 'rtol': 5e-4, 'atol': 5e-4}
+            self.check_double_backward_options = {
+                'eps': eps, 'rtol': 1e-3, 'atol': 1e-3}
+        else:
+            raise ValueError('invalid dtype')
```

Comment:
This PR is not responsible, but should use `assert`.
```suggestion
            assert False
```